admin/policies/V1.8.0__initial_policy_document.sql

```diff
@@ -1,46 +1,26 @@
-// https://docs.snowflake.com/en/user-guide/security-column-ddm-use
--- use role masking_admin;
--- USE DATABASE synapse_data_warehouse;
--- USE SCHEMA synapse;
--- CREATE MASKING POLICY IF NOT EXISTS email_mask AS (val string) returns string ->
---   CASE
---     WHEN current_role() IN ('SYSADMIN') THEN VAL
---     ELSE regexp_replace(val,'.+\@','*****@') -- leave email domain unmasked
---   END;
+USE SCHEMA POLICY_DB.PUBLIC;
 
--- ALTER MASKING POLICY email_mask SET BODY ->
---   CASE
---     WHEN current_role() IN ('SYSADMIN') THEN VAL
---     ELSE '*****'
---   END;
-
-USE ROLE SYSADMIN;
```

Comment:
Just a qq, don't we need to assume a role to implement the changes anymore?

Reply:
@danlu1 Good question. These are account policies, which are always owned by ACCOUNTADMIN (this is the only role which can create policies by default). So we specify the role which runs the policy scripts as part of the CI, as you can see [here](https://github.com/Sage-Bionetworks/snowflake/pull/148/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR152).

Reply:
Gotcha. Thanks for clarifying!